schema/home/context.js

```diff
@@ -1,5 +1,6 @@
 import { create, assign } from "lodash"
 import gravity from "lib/loaders/gravity"
+import artistLoader from "lib/loaders/per_type"
```

Comment:
Addressed with 891550d. I'm confused about why/how one is able to take in these four arguments in seemingly all situations. Is it like global context or something?